admin/static/js/reql_docs.js

```diff
@@ -2,2420 +2,2433 @@
 // hash: "186fb20ea9911710e910acfab0f4f221d59d7c04"
```

Comment:
I don't see the documentation of matchAll in that branch. Did you edit this file manually?